packages/web/src/shared/bibleTranslationClient.ts

```diff
@@ -61,6 +68,7 @@ class BibleTranslationClient {
         continue;
       }
     }
+    // Must return null, not undefined, so that this will work with useQuery
```

Comment:
Can you explain why this is the case?

Reply:
![image](https://github.com/arrocke/gloss-translation/assets/17209828/0331e8f9-8a3c-491a-952e-6cf7bdf533c8)

Reply:
Ah, good to know. That helped me find [this](https://tanstack.com/query/v4/docs/react/guides/migrating-to-react-query-4#undefined-is-an-illegal-cache-value-for-successful-queries) in their docs

Reply:
I also need to do some more reading on my own on how typescript thinks about null vs undefined. At my job, we just avoid using null if at all possible and I'm not sure that's right

Reply:
Interesting. Personally, I avoid `undefined` as much as possible, and find `null` useful, as long as you're careful not to overuse it.